libfastpm/thermalvelocity.c

```diff
@@ -375,12 +375,26 @@ fastpm_split_ncdm(FastPMncdmInitData * nid,
             //give id, mass and add thm vel
             dest->id[r] = j * src->meta._q_size + src->id[i];
             dest->mass[r] = nid->mass[j] / (nid->m_ncdm_sum / nid->n_ncdm) * M0;
-
+            
             for(d = 0; d < 3; d ++){
-                /* conjugate momentum unit [a^2 xdot, where x is comoving dist] */
+                // conjugate momentum unit [a^2 xdot, where x is comoving dist]
                 dest->v[r][d] += nid->vel[j][d] / (1. + nid->z) / HubbleConstant;
             }
             
+            //FOR TEST: change ncdm position (without this change it would be on top of cdm after 2lpt)
```

Comment:
Looks like biasing is no longer useful after you add the correct initial ncdm clustering?

Reply:
Yeah, I guess I'll delete it